### DISCUSSION TITLE
Removing the TabBar

### DISCUSSION THREAD

**Discussion by JenJell** [InfiniTimeOrg/InfiniLink#118] [Ideas]
Recently, I have been playing around with the idea of removing the tabBar from the UI. Mostly because I believe it takes up too much space with little benefit. Though, before I tried to make a pull request or go any farther with the concept, I wanted to consult others opinions on this.

If anyone wants to check out what it would look like, here is a link to my ExperimentalUI branch.
https://github.com/JenJell/InfiniLink/tree/ExperimentalUI

**Comment by liamcharger** [comment]
@JenJell I've checked out your _ExperimentalUI_ branch, and I've got to say, it's very impressive! I very much like the material backgrounds on the sheets, and I love what you're doing with live watchfaces! 

Regarding removing the _tabBar_: while I like the way it looks with no _tabBar_, I don't think putting the settings in the _DeviceView_ is a good idea. The _DeviceView_ holds everything directly related to the watch, and the settings view is a parent to everything else. What I mean to say is putting the _SettingsView_ in a list item in the _DeviceView_ feels a little out of place. Also, I've made some commits to the _main_ branch with a view that displays all the watch settings, and it won't work to have them close by each other. (I'm not sure if the watch settings view is completely necessary, so I may remove it in future commits.) 

Feedback on the rest of the _ExperimentalUI_ branch: I think the font size in the nav bars (and the WelcomeView) is a little on the small side. Also, (in the WelcomeView) I really like the idea of having a text block and a link saying "Don't have a watch? **Learn more about the PineTime**", although I think it would look better with the background image scaled up to 1.5 and the Welcome to InfiniLink text at a bolded .largeTitle. I've attached a screenshot below to give you an idea of what I mean:

<img width="289" alt="Screenshot 2024-02-11 at 11 22 59 AM" src="https://github.com/InfiniTimeOrg/InfiniLink/assets/105521623/938e9531-7202-4c4c-a3eb-b6a8f5d7685a">

I'm going to merge elements from both your _ExperimentalUI_ branch and _main_ and then create a PR with the result so you can give me your feedback. Does that sound good?

**Reply by JenJell** [reply]
Thanks for the feedback. The settings view will definitely need to be revised as I just haphazardly placed it there. Honestly, most of the stuff in the settings menu could probably be moved elsewhere, then what is left can be moved to an app debug menu. Though I will have to experiment with that later. For the welcome view, I do agree that does look better. Though, with the title and the pair button overlapping with the watch image, I will have to check how well it looks in light mode. Anyway, merging the branches together sounds good to me. I will go review the pull request and tell you if I notice anything.

**Reply by liamcharger** [reply]
I made some changes so that the text isn't overlapping the image anymore, and so it looks good in light mode now.
